test/unit/utils/test_repository.py

```diff
@@ -5,14 +5,7 @@
 
 class RepositoryTest(unittest.TestCase):
     def test_generate_repository_path(self):
-        self._compare_paths("foo", "ac/bd/foo")
-
-    def test_generate_repository_path_with_subobject(self):
-        self._compare_paths("bar/part_0001", "37/b5/bar/parts/part_0001")
-
-    def test_generate_repository_path_with_multiple_subobjects(self):
-        self._compare_paths("buz/part_0001/part_0001",
-                            "2f/bf/buz/parts/part_0001/parts/part_0001")
+        self._compare_paths("JOURNAL-ZID1234567", "4567/67/JOURNAL-ZID1234567")
```

Comment:
Ich glaube es gab da ein Missverständnis bzgl. der gewünschten Pfade. Der Pfad für `JOURNAL-ZID1234567` sollte `4500/4567/JOURNAL-ZID1234567` lauten. So ist auch das Arachne-Archiv aufgebaut.

In der jetzigen Implementation hat die zweite Ebene `67`, wenn ich das richtig sehe, keinen Nutzen, weil alle Objekte, die in `4567` landen im gleichen Unterordner landen.

Reply:
Oh stimmt, hab ich entsprechend geändert